brave/brave-core
# feat(wallet): Initial Composer UI Setup

## Description 
- Introduces the initial `Composer UI` layout
- Adds a `Send` like experience to the `Swap` screen

### Follow up work
Do to the complexity and amount of refactoring necessary, I have broken the `Composer UI` implementation out into individual issues.

- https://github.com/brave/brave-browser/issues/34471
- https://github.com/brave/brave-browser/issues/34581
- https://github.com/brave/brave-browser/issues/34582
- https://github.com/brave/brave-browser/issues/34583
- https://github.com/brave/brave-browser/issues/34585
- https://github.com/brave/brave-browser/issues/34725


Resolves <https://github.com/brave/brave-browser/issues/34305>
Resolves <https://github.com/brave/brave-browser/issues/28720>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to the `Send` and `Swap` pages in the Wallet `Page` view to see the new `Composer UI`
2. Test all functionality and make sure there are no regressions. 
3. Go to the `Send` and `Swap` pages in the Wallet `Panel` to see the new `Composer UI`
4. Test all functionality and make sure there are no regressions. 

https://github.com/brave/brave-core/assets/40611140/adbda02f-0618-444a-a7bc-32e19e319790
